scripts/testing/requirements-ci.txt

```diff
@@ -0,0 +1,5 @@
+# Requirements for top level CI operations.
+# Don't include Routemaster's own requirements here.
+pip==18.0
+tox==2.9.1
+coveralls==1.10.0
```

Comment:
If this is strictly CI, maybe put it in `.circleci/`? If it's not, maybe `requirements-test.txt`?

Reply:
Being in `.circleci` is a good call. a6dde7b